### DISCUSSION TITLE
HA Entities for Coordinator device

### DISCUSSION THREAD

**Discussion by GerdRuetten** [Koenkk/zigbee2mqtt#16925] [Devices]
Dear all,

i use z2m as add-on with Home Assistant.
@Koenkk many many thanks for this great tool!

One question:
I use in my zigbee network a SONOFF ZigBee 3.0 USB DONGLE PLUS (P-Version) flashed as coordinator and two Dongles flashed as router.
The router devices publish lqi and last seen to Home Assistant.

Is this possible for the coordinator?
I find no entitie for this device in HA.

TIA

Best regards

Gerd

**Comment by TheQue42** [comment]
I haven't got an entity for the coordinator either, (in the same setup), but why do you want one? I just learned that the LQI is calculated by the coordinator himself (and that my old conbee cheats), so there is no purpose providing the LQI value of the coordinator...

**Reply by GerdRuetten** [reply]
It's about monitoring at a central point in Home Assistant
